### PR TITLE
ENH remove inspect linkages

### LINKED ISSUE #1220: Remove conda inspect linkages calls in a mini-migrator
cc @beckermr, @erykoff

### PR DESCRIPTION
**Pull request by beckermr**
This PR removes inspect linkages in a migrator. 

Closes #1220

### CODE REVIEW

**Comment by beckermr** [comment]
Ready for a look @isuruf!

File: conda_forge_tick/migrators/cross_compile.py [hunk 1]
```diff
@@ -255,3 +255,27 @@ def migrate(self, recipe_dir: str, attrs: "AttrsTypedDict", **kwargs: Any) -> No
...
+            return True
+
+    def migrate(self, recipe_dir: str, attrs: "AttrsTypedDict", **kwargs: Any) -> None:
+        with indir(recipe_dir):
+            with open("meta.yaml") as fp:
+                meta_yaml = fp.readlines()
+
+            new_lines = []
+            for line in meta_yaml:
+                if "conda inspect linkages" in line:
```
Comment (isuruf) [inline conda_forge_tick/migrators/cross_compile.py:276]:
```suggestion
                if "conda inspect" in line:
```
for https://github.com/conda-forge/boost-cpp-feedstock/blob/master/recipe/meta.yaml#L92

File: tests/test_cross_compile.py [hunk 2]
```diff
@@ -755,6 +763,122 @@
...
+
+test:
+  requires:
+    - pytest
+    - hypothesis
+  commands:
+    - f2py -h
+    - export OPENBLAS_NUM_THREADS=1  # [unix]
+    - set OPENBLAS_NUM_THREADS=1  # [win]
+    - conda inspect linkages  # comment
```
Comment (ocefpaf) [inline tests/test_cross_compile.py:802]: We should probably add a conda inspect object here too.

Comment (beckermr) [reply tests/test_cross_compile.py:802]: done!

File: conda_forge_tick/migrators/cross_compile.py (hunk 1 above)
Comment (beckermr) [reply conda_forge_tick/migrators/cross_compile.py]: done!

**Comment by beckermr** [comment]
Ready for another look @isuruf @ocefpaf !

**Review by ocefpaf — APPROVED** [review]

**Comment by beckermr** [comment]
@CJ-Wright Should we add this to all migrations too?

**Pull request merged**